chiselFrontend/src/main/scala/Chisel/internal/Builder.scala

```diff
@@ -92,7 +92,7 @@ private[Chisel] class DynamicContext {
 
 private[Chisel] object Builder {
   // All global mutable state must be referenced via dynamicContextVar!!
-  private val dynamicContextVar = new DynamicVariable[Option[DynamicContext]](None)
+  private val dynamicContextVar = new DynamicVariable[Option[DynamicContext]](Some(new DynamicContext))
 
   def dynamicContext: DynamicContext =
```

Comment:
This seems to be a non trivial change. I think this should be raised as a separate issue.